annotation-processor/pom.xml

```diff
@@ -23,7 +23,7 @@
         <!-- This is a publicly distributed module that should be published: -->
         <deploy.skip>false</deploy.skip>
         <hibernate-validator-parent.path>..</hibernate-validator-parent.path>
-        <automatic.module.name>org.hibernate.validator.annotationprocessor</automatic.module.name>
+        <java.module.name>org.hibernate.validator.annotationprocessor</java.module.name>
```

Comment:
I'm really not sure it makes sense to put an annotation processor in the module path but... :shrug:

Reply:
I'm even less sure than that 😄. But as it had an automatic module name...